R/zzz.R

```diff
@@ -116,6 +116,20 @@ move_env_elements(RPolarsExpr, pl, c("lit"), remove = FALSE)
 
 
 .onLoad = function(libname, pkgname) {
+  # set options: this has to be done first because functions in the "pl"
+  # namespace will validate options internally
+  options(
+    polars.debug_polars = FALSE,
+    polars.df_knitr_print = "auto",
+    polars.do_not_repeat_call = FALSE,
+    polars.int64_conversion = "double",
+    polars.maintain_order = FALSE,
+    polars.no_messages = FALSE,
+    polars.rpool_active = unwrap(get_global_rpool_cap())$active,
+    polars.rpool_cap = unwrap(get_global_rpool_cap())$capacity,
+    polars.strictly_immutable = TRUE
+  )
```

Comment:
Wouldn't doing this here overwrite any options the user has set in Rprofile, etc.?

Reply:
Good point, I didn't think about that (I barely use Rprofile, and I think it's a bad idea to store options that modify so much the behavior there).

Do you know how other packages handle this?

Reply:
In any case, the value for `rpool_cap` is determined on the Rust side, so we shouldn't take into account what the user put in their Rprofile. For the rest, I'm exploring a bit more

Reply:
> Do you know how other packages handle this?

I've seen the readr package have functions that wrap options.
Here. https://github.com/tidyverse/readr/blob/e529cb2775f1b52a0dfa30dabc9f8e0014aa77e6/R/utils.R#L24-L32

Reply:
I think [7b7d7f3](https://github.com/pola-rs/r-polars/pull/726/commits/7b7d7f3c6fdfc8b184664061ecb96f12baf151f3) fixed it. I tried to set options in .Rprofile in another project and it correctly got their values on startup. I don't really know how to properly test this though, there's no `withr::with_rprofile()`...